front_camera/front_camera/frame_timing_calculator.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, video_path, bounding_box_width=300, bounding_box_height=600):
         self.video_path = video_path
         self.bounding_box_width = bounding_box_width
         self.bounding_box_height = bounding_box_height
-        self.yellow_rectangle_detector = YellowRectangleDetector()
+        self.yellow_rectangle_detector = ColorRectangleDetector("yellow")
```

Comment:
例外のキャッチ+色は大文字？（yellow→YELLOW）
```suggestion
        try:
            self.yellow_rectangle_detector = ColorRectangleDetector("YELLOW")
        except ValueError as e:
            print(e)
```